mmda/types/annotation.py

```diff
@@ -97,7 +126,9 @@ def __getitem__(self, key: int):
 
     def __deepcopy__(self, memo):
         box_group = BoxGroup(
-            id=self.id, boxes=deepcopy(self.boxes, memo), type=self.type, uuid=self.uuid
+            boxes=deepcopy(self.boxes, memo),
```

Comment:
a lot of performance issues were resolved by removing deepcopy from other places where annotation() is happening; we may not want to introduce back.  is this critical?

Reply:
This just implements support for `deepcopy`; as long as no code calls it over a span group/box group, we should be good.  

We could perhaps raise a warning here that lets user know that there's a perf penalty associated with it?

Reply:
ah gotcha;  yea a warning could be good for now

Reply:
added!